packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.0.1"
+  changes:
+    - description: Change default to use journald input for SLES 15 SP6.
+      type: enhancement
```

To me this looks to be a bugfix, since we don't support logfile for `sles` since the very beginning. Just that we had not added that to our config.
```suggestion
      type: bugfix
```